test/common/config.js

```diff
@@ -41,7 +41,6 @@ export default function () {
         )}`,
         `--elasticsearch.username=${kibanaServerTestUser.username}`,
         `--elasticsearch.password=${kibanaServerTestUser.password}`,
-        `--home.disableWelcomeScreen=true`,
```

Comment:
I believe we can remove this configuration now. We now have some logic that sets the localStorage value in the FTR when attempting to login to Kibana to handle this case in Cloud where we can't set this config value. This makes this unnecessary now.